### PR TITLE
🚀Preview Release

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Release Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

- [x] The **_base(target)_** branch for this pull request is a **_release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: release/v<major>.<minor>.<patch>
      Example: release/v1.2.3
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_release_** branch with the correct naming syntax in the following example.
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: release/v<major>.<minor>.<patch>
    Example: release/v1.2.3
    ```
  </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title is set to _**🚀Preview Release**_.

- [x] A `🚀Preview Release` label has been added to this pull request. 

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] A QA issue has been created and manual QA testing has been performed for the changes being released _(if applicable)_.

### CODE REVIEW

File: .vscode/release-notes.code-snippets [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	},
+	"Repo URL": {
+		"scope": "markdown",
+		"prefix": "repo-url",
+		"body": [
+			"[${1|BranchValidator,CASL,CICD,GitHubData,GitHubReleaseChecker,GotNuget,KDActionUtils,TagVerifier,Velaptor,VersionMiner,VersionValidator|}](https://github.com/KinsonDigital/${1|BranchValidator,CASL,GitHubData,GitHubReleaseChecker,GotNuget,KDActionUtils,TagVerifier,Velaptor,VersionMiner,VersionValidator|})"
+		],
+		"description": "Outside contributor link"
+	}	
+}
```
Comment (KinsonDigitalAdmin) [inline .vscode/release-notes.code-snippets:51]: Add single empty line to end of the file

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md [hunk 2]
```diff
@@ -1,17 +1,57 @@
...
+
+
+
+---
+
+<h2 style="font-weight:bold" align="center">Internal Changes ⚙️</h2>
+<h5 align="center">(Changes that do not affect users.  Not breaking changes, new features, or bug fixes.)</h5>
+
+1. [#116](https://github.com/KinsonDigital/CASL/issues/116) - Updated the project dotnet and language versions.
+   > **💡** Things comes with lots of performance improvements due to dotnet _**v7.0**_
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md:33]: Grammar
```suggestion
   > **💡** This comes with many performance improvements due to dotnet _**v7.0**_
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md [hunk 3]
```diff
@@ -1,17 +1,57 @@
...
+   - Updated dotnet from version _**v5.0**_ to _**v7.0**_
+   - Updated C# language version from version _**v9.0**_ to _**v11.0**_
+2. [#115](https://github.com/KinsonDigital/CASL/issues/115) - Updated the build system from standard workflows to the [KinsonDigital.CICD](https://kinsondigital.cicd) build system.
+---
+
+<h2 style="font-weight:bold" align="center">Nuget/Library Updates 📦</h2>
+
+1. [#117](https://github.com/KinsonDigital/CASL/issues/117) - Updated the following NuGet packages.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md:41]: Grammar
```suggestion
1. [#117](https://github.com/KinsonDigital/CASL/issues/117) - Updated the following NuGet packages:
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md [hunk 4]
```diff
@@ -1,17 +1,57 @@
...
+   - Updated the NuGet package **System.IO.Abstractions** from version _**v13.2.33**_ to _**v19.1.5**_
+   - Updated the NuGet package **Microsoft.CodeAnalysis.NetAnalyzers** from version _**v5.0.3**_ to _**v7.0.0**_
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md:43]: Grammar
```suggestion
   - **System.IO.Abstractions** from version _**v13.2.33**_ to _**v19.1.5**_
   - **Microsoft.CodeAnalysis.NetAnalyzers** from version _**v5.0.3**_ to _**v7.0.0**_
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by kselena — APPROVED** [review]

**Pull request merged**